tests/unit/src/main/scala/tests/TestingServer.scala

```diff
@@ -214,6 +199,18 @@ final class TestingServer(
       expectedDiff
     )
   }
+
+  def assertReferenceDiff(
+      filename: String,
+      substring: String,
+      expectedDiff: String
+  )(implicit loc: munit.Location): Unit = {
+    Assertions.assertNoDiff(
+      references(filename, substring),
```

Comment:
You can sort the results here, so that the tests don't fail.

Reply:
it returns a String :/